src/components/WatchVideo.vue

```diff
@@ -14,7 +14,7 @@
     <div v-if="video && !isEmbed" class="uk-container uk-container-xlarge">
         <ErrorHandler v-if="video && video.error" :message="video.message" :error="video.error" />
 
-        <div v-show="!video.error">
+        <div v-show="!video.error" id="video-and-info">
```

Comment:
Instead of using id, and document.getElement... You should use `ref="name"` and `this.$refs.name`